src/pages/Polling.js

```diff
@@ -277,7 +277,7 @@ class Polling extends React.Component {
   }
 
   updateVotedPollOption = async () => {
-    if (!this.props.poll) return null;
+    if (!this.props.poll || !this.state.activeAccount) return null;
```

Comment:
@b-pmcg if I didn't have this check, i'd get an error if I landed on a specific poll details page directly. feel free to remove this if you had other plans.

Reply:
ah, I've been occasionally hitting errors for this too, but it wasn't consistent, so hard to track down the source. I've sprinkled a few `if (...) return null;` in there to prevent errors, do you think that's an anti pattern? Is there a place further up the stack to ensure data is available before rendering, or is it ok to drop these types of checks where the errors occur? Not sure if there is a right or wrong answer here, just kind of an open ended question 🤔